config/CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ elseif(EXISTS ${PROJECT_SOURCE_DIR}/.git)
 
     # Create a update_version_stamp target to update the version during build.
     add_custom_target(update_version_stamp ALL
-      COMMAND ${PROJECT_SOURCE_DIR}/scripts/build/update_version_stamp.sh
+      COMMAND lua ${PROJECT_SOURCE_DIR}/scripts/build/update_version_stamp.lua
```

Comment:
No need for a new subdirectory,it can go directly in scripts/.

Reply:
I've thought it would be a good idea starting to structure them (less clobbered).

Reply:
Yeah, that's always the reason people create subdirectories :) It's really unnecessary.

Reply:
But it's nice to e.g. discover "scripts/vim-patch.sh" more easily when tab-completing scripts.
Anyway, I can move it to the others.